src/huggingface_hub/README.md

```diff
@@ -236,6 +241,43 @@ Examples using the `commit` context manager:
 ...     torch.save(model.state_dict(), "model.pt")
   ```
 
+### Non-blocking behavior
+
+The pushing methods have access to a `blocking` boolean parameter to indicate whether the push should happen
+asynchronously.
+
+In order to see if the push has finished or its status code (to spot a failure), one should use the `command_queue`
+property on the `Repository` object.
+
+For example:
+
+```python
+from huggingface_hub import Repository
+
+repo = Repository("<local_folder>", clone_from="<user>/<model_name>")
+
+with repo.commit("Commit message", blocking=False):
+    # Save data
+
+last_command = repo.command_queue[-1]
+
+# Status of the push command
+last_command.status  
+# Will return the status code
+#     -> "running" will indicate the push is still ongoin
```

Comment:
I'm not sure if mixing string with numbers is a good idea. Users will need to do non-intuitive handling if they want to rely in the status.

Reply:
```suggestion
#     -> "running" will indicate the push is still ongoing
```